### DISCUSSION TITLE
winget on windows runner creates wrong PR

### DISCUSSION THREAD

**Discussion by Gjergj** [goreleaser/goreleaser#4783] [Q&A] [answered]
When the winget publisher is run on github action with windows runner creates a wrong PR because it uses windows path separator.
 [Example](https://github.com/microsoft/winget-pkgs/pull/148927)

 Is winget built to be run only from a linux runner?
 
 My usecase: need to support chocolatey and winget. Can not run chocolatey on linux runner so I'm using windows runner. The windows runner creates the windows artifacts. So I moved winget to the windows runner.

**Comment by caarlos0** [comment, marked as answer]
its a bug

thanks for reporting

see #4786

**Reply by Gjergj** [reply]
Cool, thank you!
